ara/templates/layout.html

```diff
@@ -1,120 +1,29 @@
+    <title>ARA - Ansible Run Analysis</title>
 
-        <link href="{{ url_for('static', filename='css/bootstrap.min.css') }}" rel="stylesheet">
-        <link href="{{ url_for('static', filename='css/bootstrap-theme.min.css') }}" rel="stylesheet">
-
-        <style>
-            .FAILED {
-                background-color: #a30000;
-            }
-            .SKIPPED {
-                background-color: #047a7b;
-            }
-            .OK {
-                background-color: #3e7b04;
-            }
-            .CHANGED {
-                background-color: #9d8000;
-            }
-            .UNREACHABLE {
-                background-color: #a30000;
-            }
-
-            tr.results {
-                white-space: pre-wrap;
-                overflow-wrap: break-word;
-                word-wrap: break-word;
-                word-break: break-word;
-            }
-
-            th {
-                font-size: 120%;
-            }
-
-            .label {
-                font-size: 90%;
-            }
-
-            .label.status-filter {
-                font-size: 110%;
-            }
-        </style>
-    </head>
-    <body>
-        <div class="container" style="border-bottom:1px dashed black;">
-        <nav class="navbar navbar-inverse">
-            <div class="container-fluid">
-                <div class="navbar-header">
-                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse">
-                        <span class="sr-only">Toggle navigation</span>
-                        <span class="icon-bar"></span>
-                        <span class="icon-bar"></span>
-                        <span class="icon-bar"></span>
-                    </button>
-                    <a class="navbar-brand" href="/"><strong>ARA</strong> - Ansible Run Analysis</a>
-                </div>
-                <div class="navbar-collapse collapse">
-                    <ul class="nav navbar-nav">
-                        {% if hosts %}
-                        <li class="dropdown">
-                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Hosts <span class="caret"></span></a>
-                            <ul class="dropdown-menu">
-                                {% for host in hosts %}
-                                <li><a href="/host/{{ host }}">{{ host }}</a></li>
-                                {% endfor %}
-                            </ul>
-                        </li>
-                        {% endif %}
+    <link href="{{ url_for('static', filename='css/bootstrap.min.css') }}" rel="stylesheet">
+    <link href="{{ url_for('static', filename='css/bootstrap-theme.min.css') }}" rel="stylesheet">
+    <link href="{{ url_for('static', filename='css/ara.css') }}" rel="stylesheet">
+    <script src="{{ url_for('static', filename='js/jquery-2.2.3.min.js') }}"></script>
```

Comment:
I'm not a HTML/JSS guru but I remember reading that javascript resources should be included at the end of the page so that loading them is non-blocking (or something along those lines)